src/BookNavigator/BookNavigator.js

```diff
@@ -53,14 +52,33 @@ export class BookNavigator extends LitElement {
     this.signedIn = false;
 
     // Untracked properties
-    this.fullscreenMgr = new BRFullscreenMgr();
+    this.fullscreenMgr = new BRFullscreenMgr(this.bookreader?.el);
```

Comment:
why do we need `?` in `this.bookreader` here while in some parts of the code we don't add `?`?

Reply:
hi @dualcnhq good catch, there is a timing issue as to when bookreader is available.  i will move this instantiation out of constructor and initialize when the event `BookReader:PostInit` is being handled.